EPCPortalTesting/DataHandlerModelTests.cs

```diff
@@ -0,0 +1,65 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using EPCPortalWeb;
+using EPCPortalWeb.Models;
+using System.Collections.Generic;
+
+namespace EPCPortalTesting
+{
+    [TestClass]
+    public class DataHandlerModelTests
+    {
+        DataHandlerModel dataHandlerModel = new DataHandlerModel();
+
+        List<string> MockAPIReturn = new List<string>()
+            {
+                "1 fake street",
+                "2 fake street",
+                "3 fake street",
+                "4 fake street",
+                "5 fake street"
+            };
+
+        List<string> GetListOfAddressesByPostcode(string postcode)
+        {
+            List<string> ListOfAddresses = new List<string>();
+            //API Call
+            ListOfAddresses = MockAPIReturn;
+            return ListOfAddresses;
+        }
+
+        [TestMethod]
+        public void SearchForAddresses_WhenPostcodeIsSubmitted()
+        {
+            List<string> addresses = GetListOfAddressesByPostcode("RM96BF");
+            Assert.IsNotNull(addresses);
+        }
+
+        [TestMethod]
+        public void SearchForAddresses_WHenPostcodeIsSubmitted_DoesNotReturnMoreThan10Results()
```

Comment:
As above, this test doesn't seem to be testing DataHandlerModel, so not sure if it is needed in this test class.

Reply:
Same as above comment. once there is an actual call that can be made the method will become "Live" code

Reply:
As above, I recommend creating a new class of Integration tests that test the interaction between the Model & API caller.